docs/content/functioneel/zaken_api.md

```diff
@@ -0,0 +1,104 @@
+| Provider        | [Zaakregistratiecomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-a97b6545-d5a7-485d-9b13-3ce22db5b9cf) (GEMMA2)                                                                      |
+| Consumer        | Componenten waarmee zaken behandeld worden (Zaakafhandelcomponenten; ZAC’s) en (andere) componenten die zaakgegevens raadplegen.                                                                  |
+| Informatiemodel | [RGBZ, versie 2.00.02](https://www.gemmaonline.nl/images/gemmaonline/a/a4/EgIM_20180621_-_ag_4_-_GEMMA_RGBZ_2.0_-_CONCEPT_20180613.zip) (in-ontwikkeling)                                         |
+| Specificaties   | <https://ref.tst.vng.cloud/zrc/api/v1/schema>                                                                                                                                                     |
+| Bijzonderheden  | Het RGBZ wordt in meerdere API’s uitgewerkt. De versie van het RGBZ waarnaar verwezen wordt omvat meer dan hetgeen in deze API is uitgewerkt.                                                     |
+
+Deze API omvat (vooralsnog) de resources Zaken, Rollen en Statussen.
+
+Elke resource biedt de mogelijkheid om objecten toe te voegen (POST), te wijzigen (PUT, PATCH), te verwijderen (DELETE) en te raadplegen (GET), tenzij anders vermeld.
+
+### Resource: Zaken
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                                                                                                                                                                       |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een zaak toevoegen, gegevens van een zaak wijzigen, wijzigingen aanbrengen in de relatie van een zaak tot betrokkenen en/of objecten dan wel één of meer gegevens van zaken en genoemde relaties opvragen.                                                                                                            |
+| Voorwaarde     | 1. Het zaaktype van de zaak moet aanwezig en geldig zijn (resource: zaaktypen in API: ZTC). <br>2.  Een te relateren Betrokkene moet aanwezig zijn (resource: [...] in API:     [...]). <br>3.  Een te relateren Object moet aanwezig zijn (resource: [...] in API: [...]). <br>4.  Bij het toevoegen van een zaak mag de (functionele) Identificatie meegegeven worden; deze dient uniek te zijn. |
```

Comment:
Heeft een zaak altijd een te relateren object? Of kan het ook een dienst zijn zoals de aanvraag van een paspoort? 
Of wordt hier mee bedoeld dat als verwezen wordt naar een te relateren Object dat dit dan ook moet bestaan?
In het RGBZ 2 heet de functionele identificatie zaakidentificatie. Is het niet handiger om die termen aan te houden? Dat zoekt makkelijker. Het allermooiste zou zijn hier direct verwijzen naar de online versie van RGBZ2.

Reply:
> Heeft een zaak altijd een te relateren object? Of kan het ook een dienst zijn zoals de aanvraag van een paspoort?

Een zaak kan een gerelateerd object hebben, hoeft niet. In het relateren van diensten voorziet het RGBZ niet. Wellicht dat dat mogelijk wordt met de toevoeging van 'Verzoek' waarover nu discussie gaande is. 

> Of wordt hiermee bedoeld dat als verwezen wordt naar een te relateren Object dat dit dan ook moet bestaan?

Waarmee, op welke zinsnede doel je? En ja, een te relateren object moet bestaan. 

> In het RGBZ 2 heet de functionele identificatie zaakidentificatie. Is het niet handiger om die termen aan te houden? Dat zoekt makkelijker. Het allermooiste zou zijn hier direct verwijzen naar de online versie van RGBZ2.

M.i. terecht, 'Zaakidentificatie'. Maar in de resource heet 'ie identificatie. Toch maar Zaakidentificatie? En verwijzen naar die online versie van RGBZ2, ja graag. Helaas is die er nog niet.

Reply:
'identificatie' gewijzigd in (Zaak)identificatie.
'moet aanwezig zijn' van zaaktype, betrokkene en object gewijzigd in 'moet al bestaan'.